packages/dht/src/connection/ConnectorFacade.ts

```diff
@@ -37,13 +37,13 @@ export interface DefaultConnectorFacadeConfig {
     webrtcPortRange?: PortRange
     maxMessageSize?: number
     tlsCertificate?: TlsCertificate
-    createOwnPeerDescriptor: (connectivityResponse: ConnectivityResponse) => PeerDescriptor
+    createLocalPeerDescriptor: (connectivityResponse: ConnectivityResponse) => PeerDescriptor
```

Comment:
This could be called `createPeerDescriptor` since `ITransport` has `getPeerDescriptor` for accessing the local PeerDescriptor

Reply:
or rename ITransports getter as well

Reply:
Renamed